main.go

```diff
@@ -54,6 +54,10 @@ var (
 	BuildTime = "undefined"
 )
 
+func setupDefaultConfigValues(v *viper.Viper) {
+	v.SetDefault(RehasherServicesConfigKey, []string{applicationName})
```

Comment:
When doing this, something that helps avoid later errors is to use `SetTypeByDefaultValue(true)`:
https://pkg.go.dev/github.com/spf13/viper?tab=doc#SetTypeByDefaultValue